packages/system/src/Item/api/hooks/useItem/useItem.ts

```diff
@@ -1,11 +1,22 @@
+import { useParams } from 'packages/common/src';
 import { useQuery } from 'react-query';
 import { useItemApi } from './../useItemApi';
 
-export const useItem = (itemId: string | undefined) => {
+export const useItemId = () => {
+  const { id = '' } = useParams();
+  return id;
+};
+
+export const useItem = () => {
+  const itemId = useItemId();
+  return useItemById(itemId);
+};
+
+export const useItemById = (itemId: string | undefined) => {
   const api = useItemApi();
   return useQuery(
-    api.keys.detail(itemId ?? ''),
-    () => api.get.byId(itemId ?? ''),
+    api.keys.detail(itemId || ''),
```

Comment:
oh. oops - that was unintentional, just rewrote this when adding in the option to not specify an id